### PR TITLE
Poll drafts for april 8

### PR DESCRIPTION
**Pull request by MakerJanSky**

### CODE REVIEW

File: governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding EURe (Monerium EUR emoney) to Spark Protocol on Gnosis with the parameters below:
+
+### Spark Parameters
+
+* Token: EURe
+* Token Address: [0xcB444e90D8198415266c6a2724b7900fb12FC56E](https://gnosisscan.io/address/0xcB444e90D8198415266c6a2724b7900fb12FC56E)
+* Oracle: $oracle
+* Oracle Provider: $oracle-provider
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md:36]:
```suggestion
* Oracle: [0xab70BCB260073d036d1660201e9d5405F5829b7a](https://gnosisscan.io/address/0xab70BCB260073d036d1660201e9d5405F5829b7a)
* Oracle Provider: Chainlink
```

File: governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding sxDAI (Savings xDAI) to Spark Protocol on Gnosis with the parameters below:
+
+### Spark Parameters
+
+* Token: sxDAI
+* Token Address: [0xaf204776c7245bF4147c2612BF6e5972Ee483701](https://gnosisscan.io/address/0xaf204776c7245bF4147c2612BF6e5972Ee483701)
+* Oracle: $oracle
+* Oracle Provider: $oracle-provider
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md:36]:
```suggestion
* Oracle: [0x1D0f881Ce1a646E2f27Dec3c57Fa056cB838BCC2](https://gnosisscan.io/address/0x1D0f881Ce1a646E2f27Dec3c57Fa056cB838BCC2)
* Oracle Provider: Chainlink
```

**Review by 0xldr — CHANGES_REQUESTED** [review]
See #govops on Discord where oracle data was provided.

File: governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP27) - April 8, 2024.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/c0a5cc97c92ec737a366f122b5ec4eec7123948b/MIP102/MIP102c2-Subproposals/MIP102c2-SP27.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP27)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip102c2-sp27-mip-amendment-subproposal/23758)
+* [Amendment Pull Request](https://github.com/makerdao/mips/pull/1080)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option $Minimum_Positive_Participation? AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
```
Comment (0xldr) [inline governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP27) - April 8, 2024.md:55]:
```suggestion
**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
```

File: governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP32) - April 8, 2024.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/fda479131ec335981458f7f5f22cc22dc4af7e28/MIP102/MIP102c2-Subproposals/MIP102c2-SP32.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP32)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip102c2-sp32-mip-amendment-subproposal/23756)
+* [Amendment Pull Request](https://github.com/makerdao/mips/pull/1078)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option $Minimum_Positive_Participation? AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
```
Comment (0xldr) [inline governance/polls/Ratification Poll for MIP Amendment Subproposal (MIP102c2-SP32) - April 8, 2024.md:55]:
```suggestion
**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
```

File: governance/polls/Ratification Poll for MIP102 Amendment (MIP102c2-SP33) - April 8, 2024.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+* [Canonical Proposal Version](https://github.com/makerdao/mips/blob/f9aa8d1eb400f011f4194d80a2e01320bc9bbb10/MIP102/MIP102c2-Subproposals/MIP102c2-SP33.md)
+* [Latest Proposal Version](https://mips.makerdao.com/mips/details/MIP102c2SP33)
+* [Proposal Discussion Thread](https://forum.makerdao.com/t/mip102c2-sp33-mip102-amendment/23792)
+* [Amendment Pull Request](https://github.com/makerdao/mips/pull/1064)
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation is currently set to **10,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option $Minimum_Positive_Participation? AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
```
Comment (0xldr) [inline governance/polls/Ratification Poll for MIP102 Amendment (MIP102c2-SP33) - April 8, 2024.md:59]:
```suggestion
**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 10,000 MKR, then the following actions will be taken:**
```

File: governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+---
+title: SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md:2]: Change file name as well if including this change

```suggestion
title: SparkLend Gnosis - Onboard EURe to SparkLend - April 8, 2024
```

File: governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+summary: Onboard EURe (Monerium EUR emoney) to Spark Protocol on Gnosis with included parameters.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md:3]:
```suggestion
summary: Onboard EURe (Monerium EUR emoney) to SparkLend on Gnosis with included parameters.
```

File: governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-04-08T16:00:00
+end_date: 2024-04-11T16:00:00
+---
+# Poll: SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md:18]:
```suggestion
# Poll: SparkLend Gnosis - Onboard EURe to SparkLend - April 8, 2024
```

File: governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, April 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding EURe (Monerium EUR emoney) to Spark Protocol on Gnosis with the parameters below:
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md:29]:
```suggestion
This poll allows the MakerDAO governance community to signal their support or opposition to adding EURe (Monerium EUR emoney) to SparkLend on Gnosis with the parameters below:
```

File: governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+* [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  4,000,000 EURe
+
+Please review the proposal [thread](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033) for more information about this proposed onboarding.
+
+A risk evaluation has been provided by BA Labs and can be found [here](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/3). 
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* EURe (Monerium EUR emoney) will be onboarded to Spark Protocol Gnosis via a future executive vote.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard EURe to Spark Protocol - April 8, 2024.md:59]:
```suggestion
* EURe (Monerium EUR emoney) will be onboarded to SparkLend Gnosis via a future executive vote.
```

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md [hunk 11]
```diff
@@ -0,0 +1,75 @@
+---
+title: SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:2]: Update file name is updating poll title

```suggestion
title: SparkLend Gnosis - Onboard USDC to SparkLend - April 8, 2024
```

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+summary: Onboard USDC ( USD//C on xDai) to Spark Protocol on Gnosis with included parameters.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:3]: 1. Where does `USD//C` come from?
2. Change `Spark Protocol` to `SparkLend`
3. Remove empty space after opening parenthesis
4. xDAI vs Gnosis Chain

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-04-08T16:00:00
+end_date: 2024-04-11T16:00:00
+---
+# Poll: SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:18]:
```suggestion
# Poll: SparkLend Gnosis - Onboard USDC to SparkLend - April 8, 2024
```

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, April 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding USDC ( USD//C on xDai) to Spark Protocol on Gnosis with the parameters below:
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:29]: see comment on line 3

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+* [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  8,000,000 USDC
+
+Please review the proposal [thread](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033) for more information about this proposed onboarding.
+
+A risk evaluation has been provided by BA Labs and can be found [here](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/3). 
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* USDC ( USD//C on xDai) will be onboarded to Spark Protocol Gnosis via a future executive vote.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:59]: see comment on lines 3 and 29

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
+---
+title: SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:2]: As before SparkLend vs Spark Protocol

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+summary: Onboard USDT (Tether USD on xDai) to Spark Protocol on Gnosis with included parameters.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:3]: As before SparkLend vs Spark Protocol

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+discussion_link: https://forum.makerdao.comhttps://github.com/MakerJanSky/community/blob/Poll-Drafts-for-April-8/governance/polls/SparkLend%20Gnosis%20-%20Onboard%20EURe%20to%20Spark%20Protocol%20-%20April%208%2C%202024.md/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:4]:
```suggestion
discussion_link: https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033
```

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md [hunk 19]
```diff
@@ -0,0 +1,75 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-04-08T16:00:00
+end_date: 2024-04-11T16:00:00
+---
+# Poll: SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:18]: As before SparkLend vs Spark Protocol

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md [hunk 20]
```diff
@@ -0,0 +1,75 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, April 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding USDT (Tether USD on xDai) to Spark Protocol on Gnosis with the parameters below:
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:29]: As before SparkLend vs Spark Protocol

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md (hunk 17 above)
Comment (0xldr) [reply governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:3]: xDAI vs Gnosis Chain

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md (hunk 20 above)
Comment (0xldr) [reply governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:29]: xDAI vs Gnosis Chain

File: governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md [hunk 21]
```diff
@@ -0,0 +1,75 @@
...
+* [Borrow cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps):  8,000,000 USDT
+
+Please review the proposal [thread](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033) for more information about this proposed onboarding.
+
+A risk evaluation has been provided by BA Labs and can be found [here](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/3). 
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* USDT (Tether USD on xDai) will be onboarded to Spark Protocol Gnosis via a future executive vote.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard USDT to Spark Protocol - April 8, 2024.md:59]: Spark Protocol vs Sparklend
xDAI vs Gnosis Chain

File: governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md [hunk 22]
```diff
@@ -0,0 +1,70 @@
+---
+title: SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md:2]: Spark Protocol vs Sparklend

File: governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md [hunk 23]
```diff
@@ -0,0 +1,70 @@
...
+summary: Onboard sxDAI (Savings xDAI) to Spark Protocol on Gnosis with included parameters.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md:3]: Spark Protocol vs Sparklend

File: governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md [hunk 24]
```diff
@@ -0,0 +1,70 @@
...
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-04-08T16:00:00
+end_date: 2024-04-11T16:00:00
+---
+# Poll: SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md:18]: Spark Protocol vs Sparklend

File: governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md [hunk 25]
```diff
@@ -0,0 +1,70 @@
...
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of Spark Protocol and the Stability Facilitators. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, April 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding sxDAI (Savings xDAI) to Spark Protocol on Gnosis with the parameters below:
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md:29]: Spark Protocol vs Sparklend

File: governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md [hunk 26]
```diff
@@ -0,0 +1,70 @@
...
+* [Supply cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps): 40,000,000 sxDAI
+
+Please review the proposal [thread](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033) for more information about this proposed onboarding.
+
+A risk evaluation has been provided by BA Labs and can be found [here](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/3). 
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* sxDAI (Savings xDAI) will be onboarded to Spark Protocol Gnosis via a future executive vote.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Onboard sxDAI to Spark Protocol - April 8, 2024.md:54]: Spark Protocol vs Sparklend

File: governance/polls/SparkLend Gnosis - Update Existing Market Parameters - April 8, 2024.md [hunk 27]
```diff
@@ -0,0 +1,65 @@
...
+
+### xDAI Interest Rate Model
+
+- Decrease Max LTV from 70% to **0%**.
+- Decrease Base borrow rate from 4.88% to **0%**.
+- Set Optimal borrow rate to **12%**.
+- Set Max borrow rate to **62%**.
+- Increase Reserve factor from 0% to **5%**.
+- Increase Supply cap from 10 million xDAI to **20 million xDAI**.
+- Increase Borrow cap from 8 million xDAI to **16 million xDAI**.
```
Comment (0xldr) [inline governance/polls/SparkLend Gnosis - Update Existing Market Parameters - April 8, 2024.md:39]:
```suggestion
- Collateral: **Yes**.
  - Decrease Max LTV from 70% to **0%**.
  - Liquidation threshold: Unchanged at **75%**.
  - Liquidation bonus: Unchanged at**5%**.
- Isolation mode: **No**.
- Efficiency mode: **No**.
- Borrowable: **Yes**.
  - Base borrow rate: Decrease Base borrow rate from 4.88% to **0%**.
  - Optimal borrow rate: **12%**.
  - Max borrow rate: **62%**.
  - Optimal Utilization: **90%**.
  - Reserve factor: **5%**.
  - Borrowable in isolation mode: **Yes**
  - Siloed borrowing: **No**
- Supply cap: **20,000,000 xDAI**
- Borrow cap: **16,000,000 xDAI**
```

**Review by 0xldr — CHANGES_REQUESTED** [review]

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md (hunk 12 above)
Comment (VoteWizard) [reply governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:3]: I believe `USD//C on xDai` comes from the Gnosis chain explorer name: https://gnosisscan.io/token/0xddafbb505ad214d7b80b1f830fccc89b60fb7a83
This name might be a bit confusing

Comment (MakerJanSky) [reply governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md:3]: Correct, VoteWizard. I think `Circle USDC` and `Tether USDT` will be the updated long names unless you disagree.

**Comment by VoteWizard** [comment]
Good job @MakerJanSky and @0xldr, I've reviewed the PR and I concur with ldr's findings. I didn't find any additional issues beyond what ldr has already pointed out. Once those changes are made, it looks good to me

File: governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md (hunk 12 above)
Comment (VoteWizard) [reply governance/polls/SparkLend Gnosis - Onboard USDC to Spark Protocol - April 8, 2024.md]: `Circle USDC` and `Tether USDT` look good to me

**Review by 0xldr — APPROVED** [review]
All looks good, thanks JS

**Pull request merged**